src/celengine/category.cpp

```diff
@@ -1,11 +1,15 @@
 #include <iostream>
+#include <libintl.h>
```

Comment:
Please use `#include <celutil/util.h>` instead. It has some macro required for windows.